### PR TITLE
fix(src): Delete exactlyOne validation in scheme 

### PR DESCRIPTION
**Pull request by silanpa**
fix(src): Delete exactlyOne validation in scheme of Notification Destination

# Description

An error occurs when not supplying a block for either auth_basic or auth_token in the newrelic_notification_destination resource, even when auth is not required.

Fixes # (issue)

#1963 newrelic_notification_destination requires an authentication type even if no auth is required

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

Please delete options that are not relevant.

- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes. Go [here](https://github.com/newrelic/terraform-provider-newrelic#testing) for instructions on running tests locally.

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

**Review by sanderblue — CHANGES_REQUESTED** [review]
Hi @silanpa, thanks for the PR! Had one comment/request.

File: newrelic/resource_newrelic_notifications_destination.go [hunk 1]
```diff
@@ -50,12 +50,11 @@ func resourceNewRelicNotificationDestination() *schema.Resource {
...
-				Optional:     true,
-				MinItems:     1,
-				MaxItems:     1,
-				ExactlyOneOf: []string{"auth_basic", "auth_token"},
-				Description:  "Basic username and password authentication credentials.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				MinItems:    1,
+				MaxItems:    1,
+				Description: "Basic username and password authentication credentials.",
```
[the remark this answers is not in the document]
Comment (silanpa) [reply newrelic/resource_newrelic_notifications_destination.go:57]: Hi @sanderblue . Thank you. That is exactly what I needed. I add it now!

**Review by sanderblue — APPROVED** [review]
Awesome, thanks! 👍

**Comment by sanderblue** [comment]
@silanpa First time contributors need to sign the CLA which you can do by clicking the [yellow button in the comments to e-sign the **CLA** ](https://github.com/newrelic/terraform-provider-newrelic/pull/1972#issuecomment-1233114581)(it's just a one-time thing). Then we can merge the pull request. Thanks 🙂

**Comment by silanpa** [comment]
> @silanpa First time contributors need to sign the CLA which you can do by clicking the [yellow button in the comments to e-sign the **CLA** ](https://github.com/newrelic/terraform-provider-newrelic/pull/1972#issuecomment-1233114581)(it's just a one-time thing). Then we can merge the pull request. Thanks 🙂

Ready!!

**Comment by kidk** [comment]
Thanks @silanpa 🔥

**Pull request merged**